src/provider/WalletLinkProvider.ts

```diff
@@ -530,9 +530,6 @@ export class WalletLinkProvider
     this._addresses = newAddresses
     this.emit("accountsChanged", this._addresses)
     this._storage.setItem(LOCAL_STORAGE_ADDRESSES_KEY, newAddresses.join(" "))
-    window.dispatchEvent(
```

Comment:
Is this not needed anymore? How does this affect the wallet products that use this provider?

Reply:
this was used by the sample project only, under `chrome` folder.